.travis.yml

```
@@ -21,13 +21,14 @@ jobs:
         - flake8 diffkemp --exclude=llreve --ignore=F403,F405,W504
         - flake8 tests
         - tools/check-clang-format.sh -d
-    - env: LLVM_VERSION=5.0
+    - env: LLVM_VERSION=5.0 CC=/usr/bin/gcc CXX=/usr/bin/g++
       if: type = cron
-    - env: LLVM_VERSION=6.0
-    - env: LLVM_VERSION=7
-    - env: LLVM_VERSION=8
-    - env: LLVM_VERSION=9
-    - env: LLVM_VERSION=10
+    - env: LLVM_VERSION=6.0 CC=/usr/bin/gcc CXX=/usr/bin/g++
```

```suggestion
    - env: LLVM_VERSION=6.0 CC=/usr/bin/gcc CXX=/usr/bin/g++
      if: type = cron
```
Travis runs 5 parallel jobs, so we keep the 5 most recent versions in normal builds (PR/branch) and the rest is run in Cron builds only.

---

Wouldn't it be better to make the LLVM 10 GCC build cron-only then? In reality, the compiler change should create no meaningful difference. I would say therefore say that it is more important to run tests on multiple LLVM versions (especially since there were quite a few problems caused by LLVM version compatibility already).

---

I don't think that this actually matters. LLVM 5 versions back is already quite old, so supporting it is not crucial. Truth is that changing a compiler should not make any difference, especially to the tests, however if it ever does, we'll spot it quickly if we run the same version with both GCC and Clang. And cron is run every day, so regressions will be seen quickly anyway.

---

Alright, I have set LLVM 6.0 to cron-only.